org.openhab.binding.zigbee/src/main/java/org/openhab/binding/zigbee/internal/converter/ZigBeeConverterMeasurementRmsCurrent.java

```diff
@@ -0,0 +1,143 @@
+import com.zsmartsystems.zigbee.ZigBeeEndpoint;
+import com.zsmartsystems.zigbee.zcl.ZclAttribute;
+import com.zsmartsystems.zigbee.zcl.ZclAttributeListener;
+import com.zsmartsystems.zigbee.zcl.clusters.ZclElectricalMeasurementCluster;
+import com.zsmartsystems.zigbee.zcl.clusters.ZclOnOffCluster;
+import com.zsmartsystems.zigbee.zcl.protocol.ZclClusterType;
+
+/**
+ * ZigBee channel converter for RMS current measurement
+ *
+ * @author Chris Jackson - Initial Contribution
+ *
+ */
+public class ZigBeeConverterMeasurementRmsCurrent extends ZigBeeBaseChannelConverter implements ZclAttributeListener {
+    private Logger logger = LoggerFactory.getLogger(ZigBeeConverterMeasurementRmsCurrent.class);
+
+    private ZclElectricalMeasurementCluster clusterMeasurement;
+
+    private Integer divisor;
+    private Integer multiplier;
+
+    @Override
+    public boolean initializeConverter() {
+        logger.debug("{}: Initialising electrical measurement cluster", endpoint.getIeeeAddress());
+
+        clusterMeasurement = (ZclElectricalMeasurementCluster) endpoint
+                .getInputCluster(ZclElectricalMeasurementCluster.CLUSTER_ID);
+        if (clusterMeasurement == null) {
+            logger.error("{}: Error opening electrical measurement cluster", endpoint.getIeeeAddress());
+            return false;
+        }
+
+        try {
+            CommandResult bindResponse = clusterMeasurement.bind().get();
+            if (bindResponse.isSuccess()) {
+                ZclAttribute attribute = clusterMeasurement
+                        .getAttribute(ZclElectricalMeasurementCluster.ATTR_RMSCURRENT);
+                // Configure reporting - no faster than once per second - no slower than 10 minutes.
+                CommandResult reportingResponse = clusterMeasurement
+                        .setReporting(attribute, 3, REPORTING_PERIOD_DEFAULT_MAX, 1).get();
+                if (reportingResponse.isError()) {
+                    pollingPeriod = POLLING_PERIOD_HIGH;
+                }
+            } else {
+                pollingPeriod = POLLING_PERIOD_HIGH;
+            }
+        } catch (InterruptedException | ExecutionException e) {
+            logger.error("{}: Exception setting reporting ", endpoint.getIeeeAddress(), e);
+        }
+
+        divisor = clusterMeasurement.getAcCurrentDivisor(Long.MAX_VALUE);
+        multiplier = clusterMeasurement.getAcCurrentMultiplier(Long.MAX_VALUE);
+        if (divisor == null || multiplier == null) {
+            divisor = 1;
+            multiplier = 1;
+        }
+
+        // Add a listener
+        clusterMeasurement.addAttributeListener(this);
+
+        return true;
+    }
+
+    @Override
+    public void disposeConverter() {
+        logger.debug("{}: Closing electrical measurement cluster", endpoint.getIeeeAddress());
+
+        clusterMeasurement.removeAttributeListener(this);
+    }
+
+    @Override
+    public void handleRefresh() {
+        clusterMeasurement.getRmsVoltage(0);
```

Comment:
I think that this should be `getRmsCurrent` (as this is in the converter for RMS current).